src/complete.js

```diff
@@ -40,7 +40,7 @@ function checkConsent(cmp, store) {
 				handleConsentResult(cmp, store, vendorList);
 			}, 100);
 
-			cmp('getVendorConsents', null, vendorConsents => {
+			cmp('getConsentFields', null, vendorConsents => {
```

Comment:
ta zmiana jest konieczna ze względu na to że cmp to tutaj nasza funkcja z inita:
 `cmpFunction = function (dlApi, command, params, callback) {
        if (decoratedCommandsMap.hasOwnProperty(command)) {
            decoratedCommandsMap[command](dlApi, params, callback);
        } else {
            dlApi.cmpInterface(command, params, callback);
        }
    }`